wildmatch.go

```diff
@@ -200,8 +224,26 @@ func (w *Wildmatch) consume(t string) ([]string, bool) {
 		t = strings.ToLower(t)
 	}
 
+	var isDir bool
+	if opt.IsDirectory {
+		isDir = true
+		// Standardize the formation of subject string so directories always
+		// end with '/'
+		if !strings.HasSuffix(t, "/") {
+			fmt.Println(t)
+			t = t + "/"
+			fmt.Println(t)
```

Comment:
It looks like you have some debugging information left over in a couple of lines here.

Reply:
Oops, I forgot that. 👍